packages/loader/container-loader/src/container.ts

```diff
@@ -744,10 +744,10 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             this.logger.sendErrorEvent({ eventName: "ContainerCloseException"}, exception);
         }
 
-        this.logger.sendErrorEvent(
+        this.logger.sendTelemetryEvent(
             {
                 eventName: "ContainerClose",
-                lastSequenceNumber: this._deltaManager.lastSequenceNumber,
```

Comment:
Well `this._deltaManager.lastSequenceNumber` isn't included in the `ChildLogger` list that I see so if it would be broadly applicable (I'm not familiar with the nuance of the different sequence numbers on there) then add it there.

Reply:
Oh never mind I see your comment about dmLastMsqSeqNumber

Reply:
The only case where it would be different from dmLastMsqSeqNumber is when we did not process any op. In such case it's fine not to log it (we will have dmLastMsqSeqNumber === undefined) - we can always get snapshot sequence number from couple other places (including Container:Load_end event on successful path, but also ODSP driver will put couple events with that sequence number).
I do not think it's worth logging 99.9% identical fields in such case